### DISCUSSION TITLE
I'd like to show left Navbar allways

### DISCUSSION THREAD

**Discussion by leo-esaki** [gethyas/hyas#388] [Q&A] [answered]
I'm building an application using @hyas/doks.
On default, we can see left Navbar only in content pages, but I would like make the app shows the Navbar including on landing page always.
How can I implement that?
I need your help urgently.
like this:
![image](https://user-images.githubusercontent.com/16341483/233156803-c2db00aa-d8c5-44ea-af46-db18690427fa.png)

**Comment by h-enk** [comment, marked as answer]
Duplicate of https://github.com/h-enk/doks/discussions/1046

**Comment by leo-esaki** [comment]
Thanks Henk
